apis/r/DESCRIPTION

```diff
@@ -34,7 +34,7 @@ Imports:
     Matrix,
     stats,
     bit64,
-    tiledb (>= 0.25.0),
+    tiledb (>= 0.26.0),
```

Comment:
It may make sense to put more than one inequivality here as we know that a future (higher) tiledb (R) package may bring an unsuitable (newer) TileDB Embedded version.

Reply:
@eddelbuettel thanks! For now -- on `main` -- there is no higher version -- 0.26.0 just came into existence this week. And for "beyond that" we do indeed have a WIP going here https://github.com/single-cell-data/TileDB-SOMA/pull/2374 and once that WIP is fully fleshed out and proceduralized, watch for further changes upcoming.